utils/json/src/bigint.rs

```diff
@@ -48,3 +51,30 @@ pub mod json {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use quickcheck_macros::quickcheck;
+
+    macro_rules! to_string_with {
+        ($obj:expr, $serializer:path) => {{
+            let mut writer = Vec::new();
+            $serializer($obj, &mut serde_json::ser::Serializer::new(&mut writer)).unwrap();
+            String::from_utf8(writer).unwrap()
+        }};
+    }
+
+    macro_rules! from_str_with {
+        ($str:expr, $deserializer:path) => {
+            $deserializer(&mut serde_json::de::Deserializer::from_str($str)).unwrap()
+        };
+    }
```

Comment:
Re-declaring those macros for every test is a bit counter-productive (like https://github.com/ChainSafe/forest/pull/1992/files#diff-b734c251f7cad8d229fbb92930569fe8b759276951e060512b2fc2d5c25f540aR583-R595 or https://github.com/ChainSafe/forest/pull/1991/files#diff-1b7afae453511db460dc76eb895dfcf001193d438377d57b142bc3c695c90b8cR145-R157). A better approach would be to export it to e.g. `forest-test-utils` and then just import it. Otherwise, we're not really gaining anything from a single-use macros.